tests/x11/remote_desktop/windows_network_setup.pm

```diff
@@ -1,39 +1,41 @@
 # SUSE's openQA tests
 #
-# Copyright 2018-2021 SUSE LLC
+# Copyright 2018-2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 #
 # Summary: Remote Login: Windows access openSUSE/SLE over RDP
 # Maintainer: GraceWang <gwang@suse.com>
 # Tags: tc#1610388
 
-use strict;
-use warnings;
-use base 'x11test';
+use Mojo::Base qw(windowsbasetest);
 use testapi;
-use lockapi;
+use version_utils qw(is_sle is_opensuse);
+
+sub approve_network_popup {
+    # if there is the networks popup that ask's if it's ok to be discoverable, approve it
+    wait_still_screen stilltime => 5, timeout => 10;
```

Comment:
There should not be a need for a `wait_still_screen` as the subsequent `assert_screen` should already cover the waiting period with the sufficient timeout

Reply:
Then you might need to reconsider your needles that you use in the next call. Just think how a human tester would conduct this and we should realize we shouldn't need this command here :)